src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```
use clear naming for variables